code/game/jobs/job/upp.dm

```diff
@@ -111,7 +111,7 @@
 	skills_type = /datum/skills/SL/upp
 	equipment = TRUE
 
-/datum/job/upp/standard/generate_equipment(mob/living/carbon/human/H)
+/datum/job/upp/leader/generate_equipment(mob/living/carbon/human/H)
```

Comment:
Please answer instead of resolving it. I still don't know if this is intended or not.
The reviewers can always resolve the conversations once their review is over.

Reply:
Moved it elsewhere.